snuba/datasets/transactions_processor.py

```diff
@@ -39,6 +41,18 @@ class TransactionsMessageProcessor(MessageProcessor):
         "sentry:dist",
     }
 
+    def __merge_nested_field(self, keys: Sequence[str], values: Sequence[str]) -> str:
+        def escape_field(field: str) -> str:
+            """
+            We have ':' in our tag names. Also we may have '|'. This escapes : and \ so
```

Comment:
Is there no other character we can choose that won't be valid in a tag key or value?

Reply:
no, we just need to escape those that can be used as delimiters